inventory/hosts

```diff
@@ -1,24 +0,0 @@
-[local]
```

Comment:
I think its worth keeping a host file with just references to local. We'll still need to be able to run ansible tasks from our local machines as part of the initial bootstrapping

Reply:
ahh yeah good point, but I was figuring we have that in the integreatly_prod/inventory/hosts instead? which is why I removed it

Reply:
Yeah it makes sense using the integreatly_prod repo but there might be a scenario where a task needs to be run that doesn't require the private repo. Don't see any harm in leaving a basic inventory in place with just local referenced

Reply:
added inventories/hosts back in.